nodes/widgets/ui_chart.html

```diff
@@ -49,6 +51,14 @@
                 : null
         }
 
+        function getSeriesTypes (xAxisType) {
```

Comment:
originally wasn't going to support `json` for `category`, but then realised it is required. Left this in place, as almost certainly in the future there will be differences here